src/Uno.UI/Microsoft/UI/Xaml/Controls/PullToRefresh/RefreshContainer/Native/NativeRefreshControl.iOS.cs

```diff
@@ -0,0 +1,23 @@
+﻿using CoreGraphics;
+using UIKit;
+
+namespace Uno.UI.Xaml.Controls;
+
+public partial class NativeRefreshControl : UIRefreshControl
+{
+	public CGPoint IndicatorOffset { get; set; }
+	
+	public override void LayoutSubviews()
+	{
+		base.LayoutSubviews();
+
+		foreach (var view in Subviews)
+		{
+			var transform = !IndicatorOffset.IsEmpty ?
+				CGAffineTransform.MakeTranslation(IndicatorOffset.X, IndicatorOffset.Y) :
+				CGAffineTransform.MakeIdentity();
+			
+			view.Transform = transform;
```

Comment:
This will conflict with `RenderTransform` defined by the user in the application.

You don't need to update the transform while swipping down/up? This would mean that you are invalidating the tree on each pointer event, is this expected?

Reply:
Was actually not needed